include/walker-dropdown.php

```diff
@@ -9,6 +9,12 @@
  * @since 1.2
  */
 class PLL_Walker_Dropdown extends Walker {
+	/**
+	 * Database fields to use.
+	 * See Walker::$db_fields
```

Comment:
Maybe use a `@see` doc tag with a link to `Walker::$db_fields`?

Reply:
Would
`@see https://developer.wordpress.org/reference/classes/walker/#properties Walker::$db_fields`
be ok for you?

Reply:
Exactly! https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/see.html#see
Don't forget that you can use a description behind the URL if necessary